docs/docs/adding-forms.md

```diff
@@ -2,7 +2,169 @@
 title: Adding forms
 ---
 
-This is a stub. Help our community expand it.
+Gatsby is built on top of React. So anything that is possible with a React form is possible in Gatsby. Let's start with the following page.
```

Comment:
I think there's React documentation that would be helpful here, see e.g. [React Forms](https://reactjs.org/docs/forms.html)

Reply:
I had a link to this at the bottom. I moved it to the top (unless you were suggesting changing it to match more closely?)